modules/channel-slack/src/backend/client.ts

```diff
@@ -90,30 +99,36 @@ export class SlackClient {
     })
 
     this.router.use(`/bots/${this.botId}/callback`, this.interactive.requestListener())
-    const publicPath = await this.router.getPublicPath()
 
-    // Bot ID is used twice, because slack must setup multiple listeners itself, so can't just be redirected
-    this.logger.info(
-      `[${this.botId}] Interactive Endpoint URL: ${publicPath.replace('BOT_ID', this.botId)}/bots/${
-        this.botId
-      }/callback`
-    )
+    await this.displayUrl('Interactive', 'callback')
   }
 
   private async _setupRealtime() {
     const discardedSubtypes = ['bot_message', 'message_deleted', 'message_changed']
-    this.rtm.on('message', async payload => {
+
+    if (this.rtm) {
+      this.listenMessages()
```

Comment:
Shouldn't this listenMessage be on this.rtm instead ? you could pass this.rtm/this.events as an argument